wagtail_airtable/management/commands/import_airtable.py

```diff
@@ -116,6 +117,30 @@ def convert_mapped_fields(self, record_fields_dict, mapped_fields_dict) -> dict:
         }
         return mapped_fields_dict
 
+    def update_model_m2m_fields(self, instance, field_name, value) -> None:
+        m2m_field = getattr(instance, field_name)
+        for m2m_value in value:
+            m2m_field.add(m2m_value)
+
+    def check_field_is_m2m(self, model, data: dict):
```

Comment:
I'm not sure this name really reflects what the function is doing - looping over fields. Could be something like `get_fields_and_m2m_status` perhaps?